### ISSUE TITLE
gsutil fails to deploy static files

### ISSUE THREAD

**Issue by JackMorganNZ** [uccser/cs-unplugged#511]
```
Traceback (most recent call last):
  File "/usr/lib/google-cloud-sdk/platform/gsutil/gsutil", line 22, in <module>
    gsutil.RunMain()
  File "/usr/lib/google-cloud-sdk/platform/gsutil/gsutil.py", line 113, in RunMain
    import gslib.__main__
  File "/usr/lib/google-cloud-sdk/platform/gsutil/gslib/__main__.py", line 39, in <module>
    import boto
  File "/usr/lib/google-cloud-sdk/platform/gsutil/third_party/boto/boto/__init__.py", line 1216, in <module>
    boto.plugin.load_plugins(config)
  File "/usr/lib/google-cloud-sdk/platform/gsutil/third_party/boto/boto/plugin.py", line 93, in load_plugins
    _import_module(file)
  File "/usr/lib/google-cloud-sdk/platform/gsutil/third_party/boto/boto/plugin.py", line 75, in _import_module
    return imp.load_module(name, file, filename, data)
  File "/usr/lib/python2.7/dist-packages/google_compute_engine/boto/compute_auth.py", line 19, in <module>
    from google_compute_engine import logger
```

The following error occurs when trying to deploy static files automatically on Travis CI (https://travis-ci.org/uccser/cs-unplugged/builds/246990556), when running the command:

```
gsutil rsync -R ./csunplugged/staticfiles/ gs://cs-unplugged-dev.appspot.com/static/
```

**Comment by ravenmaster001** [comment]
https://github.com/GoogleCloudPlatform/gsutil/issues/378

**Comment by JackMorganNZ** [comment]
Fixed in 3079e73.

**Comment by ravenmaster001** [comment]
@JackMorganNZ  was 3079e73 really the correct fix for this? Did you remove 66d1051 if it wasn't needed.

**Comment by JackMorganNZ** [comment]
Sorry, correct one is 66d10514dc93233d9688fa47a4618a86ceb665c5.
